PROCESS.md

```diff
@@ -0,0 +1,163 @@
+# JSON Schema Specification Development Process
+This document formally defines the process used to evolve the JSON Schema
+specification as of the first stable release in 2023. It applies to the core
+specification and the specifications for any dialects and vocabularies
+maintained by the JSON Schema Org. It does not necessarily apply to everything
+the JSON Schema Org maintains. For example, media type registration follows the
+IETF process. Certain components used within JSON Schema, such as Relative JSON
+Pointer, may also follow a separate process.
+
+## Canonical URLs
+There MUST be a stable canonical URL for referencing any specification that
+follows this process. If the spec is made available from any other URLs, they
+SHOULD redirect to the canonical URL. If the canonical URL is changed in the
+future, all previous canonical URLs MUST remain accessible as redirects to the
+current URL.
+
+## Mutable Spec
+There is only one version of the specification and it can be updated at any time
+as long as those changes don't violate the compatibility rules identified by the
+stability model. Bug fixes, clarifications, and other non-functional spec
+updates can be merged at any time even for stable features. A change-log blog
+post SHOULD be prepared quarterly to give visibility to any changes that were
+merged in the last quarter.
+
+## The Stability Model
+Every feature in the spec has a release status. It's either evolving, stable, or
```

Comment:
Do we want to avoid contractions like we do in specs or do we think less formal language is okay for this doc?

Reply:
Are we sure using contractions in specs is discouraged? I don't think I've ever heard that. I think the language should be as natural as possible without being ambiguous. Using natural language wherever possible makes specs easier for readers to read and understand.

Reply:
Just scanning the spec for `'s`s, it's predominantly possessives.  There are a couple `it's`s and `let's` (and even a `what's`), though.

I personally tend to steer clear of contractions in formal writing, but it doesn't appear that we as a group have been sufficiently consistent in our productions to say we have a preference.